sdk/android/skin/src/main/java/com/ooyala/android/skin/BridgeMessageBuilder.java

```diff
@@ -42,7 +42,7 @@ public static WritableMap buildTimeChangedEvent(OoyalaPlayer player) {
     }
 
     WritableArray languages = Arguments.createArray();
-    Set<String> ccLanguages = player.getAvailableClosedCaptionsLanguages();
+    Set<String> ccLanguages = player.getAvailableClosedCaptionsLanguagesNames();
```

Comment:
I'm guessing this is from a previous checkin but the function names should be getAvailableClosedCaptionsLanguageNames and getLanguageNames.  The word language shouldn't be plural.